pywinauto/controls/common_controls.py

```diff
@@ -657,6 +657,36 @@ def deselect(self):
     # Non PEP-8 alias
     Deselect = deselect
 
+    #-----------------------------------------------------------
+    def search_editor(self, friendly_class_name=""):
+        """Return the editor HwndWrapper of the item
+
+        Possible ``friendly_class_name`` values:
+
+        * ``""``  Returns most possible editor of the item
+        * ``"friendlyclassname"``  Returns editor with particular friendlyclassname
+        """
+        # If currently editing in this item or some other
+        self.listview_ctrl.type_keys("{ENTER}")
+
+        # Get list of visible controls
+        parent_dlg = self.listview_ctrl.top_level_parent()
+        list_before_click = [w.handle for w in parent_dlg.element_info.descendants() if w.visible]
+
+        # After click in the visible list should editable element appears
+        self.click_input(double=True)
+        list_after_click = [w.handle for w in parent_dlg.element_info.descendants() if w.visible]
+
+        possible_editors = set(list_after_click) - set(list_before_click)
+
+        for handle in possible_editors:
+            hwnd_frclnm = hwndwrapper.HwndWrapper(handle).friendlyclassname
+            if (friendly_class_name == "" or hwnd_frclnm == friendly_class_name):
+                return hwndwrapper.HwndWrapper(handle)
+        raise RuntimeError("Element (" + friendly_class_name + ") not found in this list view item")
+    # Non PEP-8 alias
+    SearchEditor = search_editor

```

Comment:
It's a new method. We don't need this alias because there is no point in backward compatibility. :)